lib/read.js

```diff
@@ -335,6 +356,13 @@ class ReadSql extends AdapterRead {
                     readEnd: this.timeField ? to : new JuttleMoment(Infinity)
                 };
             }
+            
+            if (this.sortFields) {
```

Comment:
You mean using offset? Yes but using offset when data could be coming in in real time could cause errors in the pagination. this method takes real time data into account so IMO it's a superior pagination method.